packages/cog/src/cog.vrt.ts

```diff
@@ -8,6 +8,9 @@ export interface VrtOptions {
     addAlpha: boolean;
     /** No need to force a reprojection to 3857 if source imagery is in 3857 */
     forceEpsg3857: boolean;
+
```

Comment:
Extract nodata from tiff

Reply:
co-opted builder.bounds and added nodata to metadata object